composer/trainer/trainer.py

```diff
@@ -1716,12 +1825,10 @@ def _train_batch(self, use_grad_scaling: bool):
         assert self._train_data_spec is not None, 'The train data spec should be set on __init__ or fit()'
 
         # Cache the device batch, because `self.state.batch` gets overridden in microbatching loop
-        # TODO: fix this name collision!
         device_batch = self.state.batch
 
         # Retry until we successfully complete training and return loss
         while True:
-            start_time = time.time()
```

Comment:
did we accidentally rm the `batch_compute_time` ? Or was that intentional?

Reply:
Intentional, it was previously used for monitored_barrier